<?xml version="1.0"?>

<launch>

    <node name="data_collector_node" type="data_collector" pkg="nearness_control" output="screen">
      <remap from="data_collector/odometry" to="ground_truth/odom"/>
      <remap from="data_collector/imu" to="imu/data"/>
      <remap from="data_collector/cmd_vel" to="cmd_vel"/>
      <remap from="data_collector/synced_odom" to="synced_odom"/>
      <remap from="data_collector/synced_cmds" to="synced_cmds"/>
    </node>

</launch>
